executor/executor.go

```diff
@@ -281,7 +281,7 @@ func (executor *Executor) execute(
 		Describe("service-name", name).
 		Describe("kind", kind)
 
-	containerName, err := executor.getContainerDetails(decision.ContainerId)
+	container, _, _, err := executor.getContainerDetails(decision.ContainerId)
```

Comment:
why did you increase number of returned values in `getContainerDetails` to just use two values?

Reply:
because it is used in multiple places and in this place i just need the container

Reply:
how come? I don't see any `getContainerDetails` related code into this pr except the function definition itself.
point me to those places, please

Reply:
not the getContainerDetails i mean the getContainerById method, but i updated it anyway to return just the container